apps/endorsement-system/src/app/modules/endorsement/endorsement.service.ts

```diff
@@ -220,13 +292,14 @@ export class EndorsementService {
               parentEndorsementList.endorsementMeta,
             ),
             bulkEndorsement: true,
+            invalidated,
           },
         }
       }),
     )
 
     return this.endorsementModel.bulkCreate(endorsements, {
-      ignoreDuplicates: true, // this ignores existing endorsements conflicts
+      ignoreDuplicates: true, // this ignores existing endorsements in this list conflicts
```

Comment:
```suggestion
      ignoreDuplicates: true, // this ignores existing endorsements conflicts in this list
```

Reply:
Nice catch, this is not very clear will go with:
`this ignores endorsements that would cause unique constraint errors when inserting into this list`